src/main/kotlin/org/ozwillo/dcimporter/model/datacore/DCModel.kt

```diff
@@ -0,0 +1,62 @@
+package org.ozwillo.dcimporter.model.datacore
+
+import com.fasterxml.jackson.annotation.JsonInclude
+import com.fasterxml.jackson.annotation.JsonProperty
+
+import java.net.URI
+import java.util.Objects
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+class DCModel : Comparable<DCModel> {
```

Comment:
This is a copy of DCModel class from ozwillo spring integration lib,  I added get/set JsonProperty annotations to have cleans keys (without :) of this object in front.

Reply:
why not a data class ?

Reply:
I have tried but i have a parsing error, similar of this error in the link https://github.com/FasterXML/jackson-databind/issues/1122